### ISSUE TITLE
Utf8 acute characters problem on pdf file

### ISSUE THREAD

**Issue by malexindigo** [puppeteer/puppeteer#4453]
### Steps to reproduce

**Tell us about your environment:**

* Puppeteer version: 1.13
* Platform / OS version: Ubuntu 16.04 (docker image)
* Node.js version: 8.16

**What steps will reproduce the problem?**
```javascript
let options = {
  path: `filename.pdf`,
  printBackground: true,
  footerTemplate: '<div>footer</div>'
};
let template = `
<html>
	<head>
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Document</title>
</head>
<body>
<style type="text/css">
.text {
  line-height: 25px;
}
</style>
  <span class="text">sesión</span>
</body>
</html>
`;

const browser = await puppeteer.launch({ args: ['--no-sandbox'] });
const page = await browser.newPage();
await page.setContent(template);
await page.pdf(options);
```
**What is the expected result?**
I expect to get a pdf with the text **sesión**

**What happens instead?**
But I get a pdf wit the text **SesiÃ³n**

I try, changing the locale on the docker image, I set the charset on the html but I stil getting **SesiÃ³n**

I try also with ```page.goto(`data:text/html,${template}`,{waitUntil: 'networkidle0'});``` but using this method I have problems on style tag.

**Comment by EmmanuelAmodu** [comment]
I think Puppeteer can not parse string 'sesión' correctly, try with 'session' it seem to work with that. Hope this helps

**Comment by JoelEinbinder** [comment]
@aslushnikov 
Maybe chrome doesn't change the charset when you document.write something with a meta tag?

**Comment by albinekb** [comment]
This gives me problem with utf8 åäö characters:
```js
  await page.setContent(html, { waitUntil: 'networkidle0' })
  const pdf = await page.pdf({ format: 'A4' })
```



This works as a workaround:
```js
  await fs.writeFile(tmpFile, html, { encoding: 'utf8' })
  await page.goto(`file://${tmpFile}`, { waitUntil: 'networkidle0' })
  const pdf = await page.pdf({ format: 'A4' })
```

**Comment by aslushnikov** [comment]
This is a dupe of #4443 that was fixed in Puppeteer 1.17.0.

@malexindigo: your snippet generates a proper PDF for me with Puppeteer 1.17.0.

@albinekb same - your `page.setContent` approach now works just fine.

Thank you for taking time to file issues!!
